<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MHBA7OI" failures="0" tests="2" name="Crm Create Account Test" time="0.029" errors="0" timestamp="2021-08-25T10:25:28 IST">
  <testcase classname="com.crm.tests.CreateAccountTestScripts" name="@BeforeClass init" time="0.001">
    <failure type="java.lang.NoClassDefFoundError" message="com/crm/data/CrmTestData">
      <![CDATA[java.lang.NoClassDefFoundError: com/crm/data/CrmTestData
at com.crm.validators.CreateAccountValidators.<init>(CreateAccountValidators.java:14)
at com.crm.tests.CreateAccountTestScripts.init(CreateAccountTestScripts.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: com.crm.data.CrmTestData
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
... 28 more
]]>
    </failure>
  </testcase> <!-- @BeforeClass init -->
  <testcase classname="com.crm.util.BaseListener" name="@AfterMethod getResult" time="-1.62986732833E9">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase classname="com.crm.util.BaseListener" name="@AfterMethod getResult" time="-1.629867328333E9">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase classname="com.crm.tests.CreateAccountTestScripts" name="createAccountWithBillingAddress on [CreateAccountTestScripts passed=0 failed=0 skipped=0]" time="0.0">
    <skipped/>
  </testcase> <!-- createAccountWithBillingAddress on [CreateAccountTestScripts passed=0 failed=0 skipped=0] -->
  <testcase classname="com.crm.tests.CreateAccountTestScripts" name="createAccountWithBillingAdressAndShippingAddress on [CreateAccountTestScripts passed=0 failed=0 skipped=0]" time="0.0">
    <skipped/>
  </testcase> <!-- createAccountWithBillingAdressAndShippingAddress on [CreateAccountTestScripts passed=0 failed=0 skipped=0] -->
</testsuite> <!-- Crm Create Account Test -->
